docs/en/Configuration-Settings.md

```diff
@@ -8,44 +8,89 @@ priority: 1
-for Alluxio in different contexts.
+Alluxio can be configured by setting the values of supported [configuration properties
+](Configuration-Parameters.html). For a user who wants to customize how an application  (e.g., a
+Spark or MapReduce job) interacting with Alluxio, check
+[how to configure Alluxio applications](#configure- applications);
+for Alluxio admin who wants to customize Alluxio service, check
+[how to configure Alluxio clusters](#configure-alluxio-cluster).
 
-## Configuration in Alluxio
+# Configure Applications
 
-Alluxio runtime respects three sources of configuration settings:
+Customizing how an application job interacts with Alluxio can be application specific. Here we
+provide recommendation for a few common applications, as well as an general solution.
 
-1. [Application settings](#application-settings). Setting Alluxio configuration in this way is application-specific,
-and is required each time when running an application instance (e.g., a Spark job).
-2. [Environment variables](#environment-variables). This is an easy and fast way to set the basic configuration
-to manage Alluxio servers and run Alluxio shell commands.
-Note that, configuration set through environment variables may not be realized by applications.
-3. [Property files](#property-files). This is a general approach to customize any
-[supported Alluxio configuration](#appendix). Configuration in those files can be respected by Alluxio servers,
-as well as applications.
+## Alluxio Shell Commands
 
-The priority to load configuration settings, from the highest to the lowest, is
-application settings (if any), environment variables, property files and the defaults.
+Alluxio shell users can use JVM system property `-Dproperty=value` to specify an Alluxio property in commandline. For example, the following Alluxio shell command sets the write type to
+`CACHE_THROUGH` when copying files to Alluxio:
+
+```bash
+$ bin/alluxio fs -Dalluxio.user.file.writetype.default=CACHE_THROUGH copyFromLocal README.md /README.md
+```
+
+
+## Spark Jobs
+
+Spark users can pass JVM system properties to Spark jobs by adding `"-Dproperty=value"` to
+`spark.executor.extraJavaOptions` for Spark executors and `spark.driver.extraJavaOptions` for
+Spark drivers. For example, the following Spark job sets write type to `CACHE_THROUGH` when
+writing to Alluxio:
+
+```bash
+$ spark-submit \
+--conf 'spark.driver.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+--conf 'spark.executor.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+...
+```
 
-### Application settings
 
-Alluxio shell users can use `-Dkey=property` to specify an Alluxio configuration value in commandline. For example,
+## Hadoop MapReduce Jobs
 
-{% include Configuration-Settings/specify-conf.md %}
+Hadoop MapReduce users can add `"-Dproperty=value"` in `hadoop jar` command-lines. For example, the
```

Comment:
This applies to `yarn jar ...` command too, right?

"in `hadoop jar` command-lines" -> "with the `hadoop jar` command."